docs/runtime/minimal-runtime.md

```diff
@@ -254,21 +264,154 @@ level=info module=net-runner caller=root.go:152 ts=2021-06-14T08:42:47.219513806
 ```
 <!-- markdownlint-enable line-length -->
 
-:::info
+:::tip
 
 The local network runner will take control of the current terminal until you
 terminate it via Ctrl+C. For the rest of the guide keep the local network
 running and use a separate terminal to run the client.
 
 :::
 
+## Testing From Oasis CLI
+
+After you have the runtime running in your local network, the next step is to
+test that it actually works. Let's see, if the local network was correctly
+initialized and the runtime is ready:
+
+```bash
+${OASIS_CORE_PATH}/oasis-node control status -a unix:/tmp/minimal-runtime-test/net-runner/network/client-0/internal.sock
```

Comment:
You should probably use the new CLI, e.g. `oasis inspect node-status` (after adding the localhost network).